default_vocab.ttl

```diff
@@ -399,6 +399,10 @@ np:Nanopublication a owl:Class;
 
 whyis:searchApi whyis:hasView "search-api.json".
 
+<searchApi2> a whyis:searchApi2 . 
```

Comment:
You should probably modify the existing API in a case like this, rather than build a new one. It should be backwards compatible, and if not, we aren't at 1.0 anyways.